source/containers/FormField/FormField.js

```diff
@@ -109,6 +110,11 @@ const inputTypes = {
   card: {
     component: DynamicCardRenderer,
   },
+  imageUploader: {
```

Comment:
Just of curiosity, do we only support images? Otherwise file as an input name would better follow standard input declaration, atleast if you look on the web :)

Reply:
This particular component only supports images, because the library used to load files; and also the way I display the images in the component will only work for images; therefore the name. We will have to build another component if we want to support other files like pdfs, or if we just want file uploads in general.